rollup.config.js

```diff
@@ -14,5 +15,12 @@ export default {
     },
   ],
   external: Object.keys(pkg.devDependencies),
-  plugins: [babel()],
+  plugins: [
+    babel({
+      babelHelpers: 'bundled',
+    }),
+    replace({
+      __DEV__: "process.env.NODE_ENV === 'development'",
```

Comment:
since `react-use-form-state` is not distributed with a minified bundle, this condition would still need to be evaluated by the consumer's bundler at the time of bundling.